src/components/Navbar.js

```diff
@@ -38,7 +39,42 @@ const Navbar = () => {
         </p>
         <BsCodeSlash color="#fff" size={33} />
       </div>
+      {localStorage.getItem('username') ?
+      <>
       <menu>
+        <ul
+          className="nav-links"
+          id={showMenu ? "nav-links-mobile" : "nav-links-mobile-hide"}
+        >
+          <li>
+            <a href="/">Home</a>
+          </li>
+          <li>
+            <a href="/users/:id">Profile</a>
```

Comment:
Instead of `:id` it should be the username that you can access from the context.
```react
<a href={`/users/${username}`}>Profile</a>
```